src/shared/components/common/modal/Modal.tsx

```
@@ -35,39 +36,51 @@ const sizeClasses = {
     }
-  }, [isOpen, close]);
+  }, [open, close]);
 
-  if (!isOpen) return null;
+  if (!open) return null;
 
   return (
     <div
       role='dialog'
       aria-modal='true'
       className='fixed inset-0 z-50 flex items-center justify-center bg-gray-900/30'
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<dialog>


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)